data/json/recipes/chem/fuel.json

```diff
@@ -73,19 +73,21 @@
     "subcategory": "CSC_CHEM_FUEL",
     "skill_used": "cooking",
     "difficulty": 6,
-    "time": 60000,
+    "result_mult": 4,
+    "time": 440000,
     "book_learn": [ [ "textbook_biodiesel", 5 ] ],
+    "batch_time_factors":[99, 1],
     "qualities":[ { "id": "CHEM", "level": 2 } ],
     "tools": [ [
       [ "fire", -1 ],
-      [ "hotplate", 20 ],
-      [ "toolset", 20 ]
+      [ "hotplate", 80 ],
```

Comment:
Arguably 100 (a stack of batteries) would be more appropriate given that recipe is likely to see considerable use in batches